apps/src/templates/studioHomepages/ResourceCard.jsx

```diff
@@ -38,23 +38,27 @@ class ResourceCard extends Component {
     } = this.props;
     const localeStyle = isRtl ? styles.rtl : styles.ltr;
 
-    const buttonStyles = [styles.button];
-    const cardStyles = [styles.card, localeStyle];
-    const titleStyles = [styles.title, localeStyle];
-    const descriptionStyles = [styles.text, styles.description, localeStyle];
+    const buttonStyles = styles.button;
+    const cardStyles = {...styles.card, ...localeStyle};
+    const titleStyles = {...styles.title, ...localeStyle};
+    const descriptionStyles = {
+      ...styles.text,
+      ...styles.description,
+      ...localeStyle
+    };
 
     if (['sm', 'xs'].includes(responsiveSize)) {
-      cardStyles.push(styles.cardSmall);
-      titleStyles.push(styles.titleSmall);
-      descriptionStyles.push(styles.descriptionSmall);
+      merge(cardStyles, styles.cardSmall);
+      merge(titleStyles, styles.titleSmall);
+      merge(descriptionStyles, styles.descriptionSmall);
     }
 
     if (allowWrap) {
-      buttonStyles.push(styles.buttonAllowWrap);
-      cardStyles.push(styles.cardAllowWrap);
-      titleStyles.push(styles.titleAllowWrap);
+      merge(buttonStyles, styles.buttonAllowWrap);
```

Comment:
Nit: I think the preference is to use something native to JS when we can -- I think you could `Object.assign(buttonStyle, styles.buttonAllowWrap)` here instead?

Reply:
Ah, good call! I get Lodash happy sometimes tehe ... will update!

Reply:
I only know this "preference" because I do the same thing and have been reminded of it as well :)